packages/components/src/query-controls/index.tsx

```diff
@@ -87,7 +119,7 @@ export default function QueryControls( {
 					selectedCategories &&
 					selectedCategories.map( ( item ) => ( {
 						id: item.id,
-						value: item.name || item.value,
+						value: item.name || item.value || '',
```

Comment:
Without this, there's a TS error:

<img width="872" alt="Screen Shot 2022-12-21 at 5 46 06 PM" src="https://user-images.githubusercontent.com/4063887/209024508-e18702e5-70d3-4d97-8475-5a146e87dad7.png">

Reply:
It's possible that `item.value` is `undefined`, and the `FormTokenField` prop `value` doesn't allow that.

Reply:
Good idea, remove the fallback in https://github.com/WordPress/gutenberg/pull/46721/commits/4e19a9ff13737b992d3b98f47fa55ee11cb85d82